code/src/components/Styling.js

```diff
@@ -0,0 +1,175 @@
+import styled, { keyframes } from 'styled-components';
```

Comment:
I really like this solution of using a separate component for some "main" styling. Must make work quicker and easier